articles/112_generated_interfaces_cpp.md

```diff
@@ -50,6 +50,40 @@ For each message two files are being generated:
 This allow to add addition files beside the one with the suffix `__struct` to provide additional functionality.
```

Comment:
Update in fc387348fe8f233bbfb4f1cb77de4aa075e78df4 - even if the line was not even part of this PR.